src/Symfony/Component/Form/Test/Traits/ValidatorExtensionTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Test\Traits;
+
+use Symfony\Component\Form\Extension\Validator\ValidatorExtension;
+use Symfony\Component\Form\Test\TypeTestCase;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+trait ValidatorExtensionTrait
+{
+    protected $validator;
+
+    protected function getValidatorExtension()
```

Comment:
The trait is now missing the tearDown (or maybe a tearDownAfterClass?) to set the validator property to `null`.

Reply:
`tearDownAfterClass` added to the TypeTestCase class

Reply:
Can't use `tearDownAfterClass` since it's static, so I added `tearDown` instead